app/components/Header/index.js

```diff
@@ -81,12 +89,43 @@ function AccountDropdownItems({
 class Header extends Component<Props, State> {
   state: State = {
     accountOpen: false,
-    shake: false
+    shake: false,
+    forgotPassword: false,
+    registerUser: false
+  };
+
+  toggleRegisterUser = (e: Event) => {
+    this.setState({ registerUser: true });
+    e.stopPropagation();
```

Comment:
why do you need `stopPropagation` here?

Reply:
toggleRegisterUser, toggleForgotPassword and toggleBack all need the stopPropagation to prevent the whole modal from closing every time the Event is triggered

Reply:
hm, that seems like a funky part of the modal, but if that's the case it's usually common to use `event.preventDefault()` instead (at the start of the function)

Reply:
nevermind, I was wrong :)